tests/test_parse/test_genbank.py

```diff
@@ -505,29 +505,37 @@ def test_init(self):
         """Location should init with 1 or 2 values, plus params."""
         l = Location(37)
         self.assertEqual(str(l), "37")
-        l = Location(37, Ambiguity=">")
+        l = Location(37, ambiguity=">")
         self.assertEqual(str(l), ">37")
-        l = Location(37, Ambiguity="<")
+        l = Location(37, ambiguity="<")
         self.assertEqual(str(l), "<37")
-        l = Location(37, Accession="AB123")
+        l = Location(37, accession="AB123")
         self.assertEqual(str(l), "AB123:37")
-        l = Location(37, Accession="AB123", Db="Kegg")
+        l = Location(37, accession="AB123", db="Kegg")
         self.assertEqual(str(l), "Kegg::AB123:37")
 
         l1 = Location(37)
         l2 = Location(42)
         l = Location([l1, l2])
         self.assertEqual(str(l), "37..42")
-        l3 = Location([l1, l2], IsBounds=True)
+        l3 = Location([l1, l2], is_bounds=True)
         self.assertEqual(str(l3), "(37.42)")
-        l4 = Location([l1, l2], IsBetween=True)
+        l4 = Location([l1, l2], is_between=True)
         self.assertEqual(str(l4), "37^42")
         l5 = Location([l4, l3])
         self.assertEqual(str(l5), "37^42..(37.42)")
-        l5 = Location([l4, l3], Strand=-1)
+        l5 = Location([l4, l3], strand=-1)
         self.assertEqual(str(l5), "complement(37^42..(37.42))")
 
 
+def test_Location_start():
+    """the start and stop should reflect 0-based indexing (python style), not 1-based indexing (genbank style).
+    This means they should be 1 less than the data given"""
+    l = Location(37)
+    assert str(l.start()) == "36"
```

Comment:
errr, why not just `l.start == 36`? Does it need to be cast to a string?

Reply:
lol, I was just copying the above tests